### DISCUSSION TITLE
Admin account does not work

### DISCUSSION THREAD

**Discussion by nvssln** [microsoft/AzUrlShortener#249] [Q&A] [answered]
Thanks for putting this solution.  While I am trying to explore, I used the single click deployment which setup all resources. 

The "adminBlazorWebsite" would not let me login.  It says: Invalid login attempt. Doesn't matter whether I specify the credentials during single click deployment or refer to the ones in solution. 

Am I missing any particular part of the step? Please let me know.

**Comment by fboucher** [comment, marked as answer]
Hey @nvssln,
When you created your password did you respect the requirement?

Password use to connect into the admin Blazor Website. It MOST have:
- an uppercase character
- a lowercase character
- a digit
- a non-alphanumeric character
- must be at least six characters long 

I'm assuming that's the problem. You can go in the Settings of the webapp (from the portal.azure.com) and change it.

**Reply by ghost** [reply]
*must

**Comment by nvssln** [comment]
That was it.  Thank you very much Frank.
